scripts/release/mule/test/tests/post/verify_genesis_file.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "[$0] Testing network string in genesis.json"
+
+# We're looking for a line that looks like the following:
+#
+#       "network": "mainnet",
+#
+
+cd "./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
+
+if [ ! -f /var/lib/algorand/genesis.json ]
+then
+    echo "[$0] The genesis file is not present."
+    exit 1
+fi
+
+if [ "$NETWORK" != "$(jq -r '.network' /var/lib/algorand/genesis.json)" ]
+then
+    echo "[$0] The network \`$NETWORK\` set in \`genesis.json\` is incorrect."
+    exit 1
+fi
+
+echo "[$0] The network \`$NETWORK\` set in \`genesis.json\` is correct."
```

Comment:
Consider rephrasing to "The network set in genesis.json is correctly set to $NETWORK".  Show full path to genesis.json file.

Reply:
I'll update to show the full path.